rust/lancedb/src/database/listing.rs

```diff
@@ -118,6 +137,12 @@ impl DatabaseOptions for ListingDatabaseOptions {
                 enable_v2_manifest_paths.to_string(),
             );
         }
+        if let Some(enable_stable_row_ids) = self.new_table_config.enable_stable_row_ids {
```

Comment:
can we also add a test at rust layer, instead of having to rely on python test to verify the functionality